mlibrary/lib.umich.edu
# Website 104 amenities order

# Overview
The visit.js template was not sorting the list items under "Amenities". 

The list entries on this template for each section have a weight assigned to them in Drupal that is set by the editor. The other sections of the template were properly sorting the entries by weight before mapping them to the UI, but Amenities was not.

There was a custom function for the field_visit property that did all of this, but nothing for the field_amenities property. I sorted and mapped the properties inline instead of passing everything to a function for field_amenities, then modified field_visit to follow the pattern and remove the redundant function HTMLList().

I also had to add the weight value to the GraphQL queries.

> This pull request fixes [WEBSITE-104](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-104).

## Testing
Visit template is all of the pages under [the Locations and Hours page](https://lib.umich.edu/locations-and-hours).

The page that was mentioned in the Jira ticket, for example, is [the Clark Commons location page](https://lib.umich.edu/locations-and-hours/clark-commons)

- Make sure the PR is consistent in these browsers:
- Run accessibility / screen reader tests:
